hack/test-cmd.sh

```diff
@@ -337,12 +330,19 @@ for test in "${tests[@]}"; do
   os::cmd::try_until_text "oc get projects -o name" "project/${namespace}"
   os::test::junit::declare_suite_end
 
-  ${test}
+  if ! ${test}; then
```

Comment:
```sh
$ cat test.sh 
#!/bin/bash

function my_trap() {
	echo "Inside of trap!"
}

trap my_trap EXIT

set -o errexit

grep

$ if ./test.sh; then echo "Success!"; else echo "Failure!"; fi
Usage: grep [OPTION]... PATTERN [FILE]...
Try 'grep --help' for more information.
Inside of trap!
Failure!
```

Seems to work fine -- I'm not familiar with the `EXIT` trap override you are talking about. Do you have a link for it?